index.html

```diff
@@ -150,6 +159,7 @@ <h2>Multi-post <br>Story</h2>
       </div>
     </section>
     <section class="about-sect">
+      <div class="arrange-desk">
       <div class="about">
         <h2 class="about-intro intro">About <br />Myself</h2>
         <p class="about-pitch">
```

Comment:
- [ ] Kindly be noticed that the left side in the about me section is wider than it is supposed to be in the desktop size.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-03 01-44-14](https://user-images.githubusercontent.com/33752735/171758442-9cfdad34-5066-4f45-bb9d-457c0adc91a6.png) | ![Screenshot from 2022-06-03 01-38-45](https://user-images.githubusercontent.com/33752735/171758458-4cd65cb6-7c46-4c3b-9345-53ca29c8169b.png)